rxlifecycle/src/main/java/com/trello/rxlifecycle/UntilCorrespondingEventObservableTransformer.java

```diff
@@ -30,11 +30,13 @@ public UntilCorrespondingEventObservableTransformer(@NonNull Observable<R> share
         return source.takeUntil(takeUntilCorrespondingEvent(sharedLifecycle, correspondingEvents));
     }
 
+    @NonNull
```

Comment:
Are all of these really necessary? The interface implements it and that's all that people interact with.